Detectors/TRD/calibration/include/TRDCalibration/CalibratorVdExB.h

```diff
@@ -30,11 +32,26 @@ namespace o2
 namespace trd
 {
 
+struct FitHelper {
+  double operator()(const double* par) const;
+  std::unique_ptr<TGraph> createTmpGraph(double vdFit, double laFit) const;
+  std::array<std::unique_ptr<TProfile>, constants::MAXCHAMBER> profiles; ///< profile histograms for each TRD chamber
```

Comment:
It seems like the FitHelper always processes one chamber at once, so why not have only one TProfile in here? Another question is why a TProfile is needed at all, it seems this is an extra copy of the data that does not add functionality.